cloudant-client/src/main/java/com/cloudant/client/org/lightcouch/MetaInformation.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright © 2018 IBM Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+package com.cloudant.client.org.lightcouch;
```

Comment:
I think this needs to be in the com.cloudant.api.model package, I don't think we consider any lightcouch classes API - with the exception of some exceptions.

Reply:
Moved to `com.cloudant.client.api.model` in ccead17 - does that make more sense?